docs/guide/core-widgets.md

```diff
@@ -85,6 +87,10 @@ Including a class with a style will not automatically apply any styles. You stil
 
 <!-- TODO: Link to how-to about configuring sanitize-html for pasting in rich text -->
 
+### Allowing links to specific piece-types
+
+By default, the rich text widget allows you to add links to URLs or internal pages. The `linkWithType` option allows you to add any links to any `piece-type` show page. Simply pass an array with the name of each desired `piece-type`. If you want to maintain linking to internal pages, also add `@apostrophecms/any-page-type` to your array. Note that the `piece-type` index page will be included in the collection retrieved by `@apostrophecms/any-page-type` and does not need to be added by default.
```

Comment:
"any links to any" -> "links to any"

Reply:
Last sentence is correct but maybe could be a little easier to follow if you're not super deep on piece pages yet... how about "Note that you don't need to change this setting if you just want to link to the main index page for a piece type."

Reply:
Where the heck did the extra "any" come from? Changed for both.